draft-packages/menu/KaizenDraft/Menu/MenuDropdown.tsx

```diff
@@ -79,23 +81,29 @@ const MenuDropdown = ({
     }
   }
 
+  useEffect(() => {
+    window.addEventListener("resize", handleDocumentResize, false)
+
+    return () => {
+      window.removeEventListener("resize", handleDocumentResize, false)
+    }
+  }, [handleDocumentResize])
+
```

Comment:
I’ve separated the two `useEffect` calls here so they can be reasoned/changed independently.